src/playback/internal/playbackcontroller.cpp

```diff
@@ -305,6 +305,10 @@ mu::notation::INotationSoloMuteState::SoloMuteState PlaybackController::trackSol
 void PlaybackController::setTrackSoloMuteState(const InstrumentTrackId& trackId,
                                                const notation::INotationSoloMuteState::SoloMuteState& state) const
```

Comment:
To fix your main build problem, you should be able to remove the const qualifier from `setTrackSoloMuteState` (I'm not sure why this was const in the first place, it's a setter after all). You’ll need to do this in a few other places too:

`playbackcontroller.h` , `iplaybackcontroller.h`, `playbackcontrollermock.h`, `playbackcontrollerstub.cpp`,      `playbackcontrollerstub.h`